### DISCUSSION TITLE
Condiciones para la base de datos.

### DISCUSSION THREAD

**Discussion by cesarparra96** [rramosp/ai4eng.v1#148] [UDEA Students]
Buena tarde.

Mi grupo de trabajo y yo queremos realizar un trabajo con un dataset que contiene un EEG, pero en las condiciones pedidas en el correo se indica que debe tener 5000 filas y 30 columnas. Nuestro data set cuenta con dimensiones de 500 x 22 x 288, donde el 500 corresponde al número de filas, el 22 al número de columnas y el 288 al número de repeticiones que se tomaron los datos.

¿Podríamos usar este data set o este no cumple con las condiciones planteadas?
En caso de poder utilizar dicho dataset, ¿Cómo hacer el 10% de las columnas como categorías?

**Comment by rramosp** [comment]
hola cesar, por su naturaleza ese dataset es un poco más complejo de lo
requerido para la materia. así que si os atravéis con el mismo adelante!!!

On Mon, Jun 27, 2022 at 5:22 PM cesarparra96 ***@***.***>
wrote:

> Buena tarde.
>
> Mi grupo de trabajo y yo queremos realizar un trabajo con un dataset que
> contiene un EEG, pero en las condiciones pedidas en el correo se indica que
> debe tener 5000 filas y 30 columnas. Nuestro data set cuenta con
> dimensiones de 500 x 22 x 288, donde el 500 corresponde al número de filas,
> el 22 al número de columnas y el 288 al número de repeticiones que se
> tomaron los datos.
>
> ¿Podríamos usar este data set o este no cumple con las condiciones
> planteadas?
> En caso de poder utilizar dicho dataset, ¿Cómo hacer el 10% de las
> columnas como categorías?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/148>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIIOPMOHRIB5BX4YJD3VRISRBANCNFSM52ACAIGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
